cmd/bosun/main.go

```diff
@@ -19,6 +16,9 @@ import (
 	"syscall"
 	"time"
 
+	version "bosun.org/_version"
```

Comment:
Why the rename of the import  ?

Reply:
Not renamed, just reordered to be in line with `goimports`

Reply:
That's `goimports` auto-formatting the imports.